deepchecks/core/common.py

```diff
@@ -0,0 +1,77 @@
+import importlib
+import typing as t
+
+from typing_extensions import Literal as L
+
+from deepchecks import __version__
+from deepchecks.utils.logger import get_logger
+
+__all__ = ['importable_name', 'import_type', 'validate_config']
+
+
+def importable_name(obj: t.Any) -> str:
+    kind = type(obj) if not isinstance(obj, type) else obj
+    name = kind.__qualname__
+    module = kind.__module__
+    return f'{name}.{module}'
+
+
+def import_type(full_name: str, base: t.Optional[t.Type[t.Any]] = None):
```

Comment:
why do we need "base" if we have full name?

Reply:
it has totally different purpose/
it helps to verify that what we had imported is actually a subtype of `BaseCheck`/`BaseSuite`